packages/framework/src/Controller/Admin/MenuController.php

```diff
@@ -4,21 +4,22 @@
 
 namespace Shopsys\FrameworkBundle\Controller\Admin;
 
-use Shopsys\FrameworkBundle\Component\Domain\DomainFacade;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
 
 class MenuController extends AdminBaseController
 {
     /**
-     * @param \Shopsys\FrameworkBundle\Component\Domain\DomainFacade $domainFacade
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
      */
-    public function __construct(protected readonly DomainFacade $domainFacade)
+    public function __construct(protected readonly Domain $domain)
     {
     }
 
     public function menuAction()
     {
         return $this->render('@ShopsysFramework/Admin/Inline/Menu/menu.html.twig', [
-            'domainConfigs' => $this->domainFacade->getAllDomainConfigs(),
+            'domainConfigs' => $this->domain->getAll(),
+            'locales' => $this->domain->getAllLocales(),
```

Comment:
I think this should use the same method as the `checkLocaleIsSupported.` It may prevent some unpleasant surprises when someone changes the behavior of only one method

Reply:
rewritten in c1495fa35e